packages/babel-preset-env/src/debug.ts

```diff
@@ -1,9 +1,6 @@
-// @flow
+import { getInclusionReasons } from "@babel/helper-compilation-targets";
 
-import {
-  getInclusionReasons,
-  type Targets,
-} from "@babel/helper-compilation-targets";
```

Comment:
it looks, I had eslint not working while rebasing… - this was to be fixed, by it… -- will update other typescript migration PRs

Reply:
We recently updated prettier, maybe you had the old version in `node_modules`

Reply:
I guess, that as well… anyway rebasing, everything after reinstalling dependencies, and building the project(to have eslint working)